artifacts/definitions.py

```diff
@@ -19,6 +19,7 @@
     'Cloud': 'Cloud applications artifacts.',
     'Cloud Storage': 'Cloud storage artifacts.',
     'Configuration Files': 'Configuration files artifacts.',
+    'DB': 'Database specific artifact.',
```

Comment:
This is very generic. What use case did you have in mind? Maybe SQLiteDB ?

Reply:
I thought about just DB because this would include all format, or else there should be one label for major DB types? I feel that could be too granular. 
My main use case comes from macOS artifacts, where most of the data from apps  are stored in sqlite db (and this applies to iOS and Android too). We could just have sqliteDB, sure.